### PR TITLE
projucer: add page

### LINKED ISSUE #7834: page request: Projucer
https://juce.com/discover/stories/projucer-manual#10.4-command-line-tools

### PR DESCRIPTION
**Pull request by ecrax**
Closes #7834

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/projucer.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# projucer
+
+> A project manager for JUCE framework applications.
+> More information: <https://juce.com/discover/stories/projucer-manual#10.4-command-line-tools>.
+
+- Display the help manual:
```
Comment (kbdharun) [inline pages/common/projucer.md:6]: Thanks for your contribution. Can you place the help command and its command description last as it is what we usually prefer?

Comment (ecrax) [reply pages/common/projucer.md]: this should be better :)

**Review by kbdharun — APPROVED** [review]

File: pages/common/projucer.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Update the version number in a project:
+
+`projucer --set-version {{version_number}} {{project_file}}`
+
+- Generate a JUCE project from a PIP file:
+
+`projucer --create-project-from-pip {{path/to/PIP}} {{path/to/output}}`
+
+- Remove all JUCE-style comments (`//=====` or `//-----` or `///////`):
```
Comment (pixelcmtd) [inline pages/common/projucer.md:22]:
```suggestion
- Remove all JUCE-style comments (`//=====`, `//-----` or `///////`):
```

File: pages/common/projucer.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# projucer
+
+> A project manager for JUCE framework applications.
+> More information: <https://juce.com/discover/stories/projucer-manual#10.4-command-line-tools>.
+
+- Display information about the project:
+
+`projucer --status`
```
Comment (pixelcmtd) [inline pages/common/projucer.md:8]:
```suggestion
- Display information about a project:

`projucer --status {{path/to/project_file}}`
```

File: pages/common/projucer.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Resave all files and resources in a project:
+
+`projucer --resave {{project_file}}`
```
Comment (pixelcmtd) [inline pages/common/projucer.md:12]:
```suggestion
`projucer --resave {{path/to/project_file}}`
```

File: pages/common/projucer.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Update the version number in a project:
+
+`projucer --set-version {{version_number}} {{project_file}}`
```
Comment (pixelcmtd) [inline pages/common/projucer.md:16]:
```suggestion
`projucer --set-version {{version_number}} {{path/to/project_file}}`
```

File: pages/common/projucer.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Generate a JUCE project from a PIP file:
+
+`projucer --create-project-from-pip {{path/to/PIP}} {{path/to/output}}`
+
+- Remove all JUCE-style comments (`//=====` or `//-----` or `///////`):
+
+`projucer --tidy-divider-comments {{target_folder}}`
```
Comment (pixelcmtd) [inline pages/common/projucer.md:24]:
```suggestion
`projucer --tidy-divider-comments {{path/to/target_folder}}`
```

**Review by pixelcmtd — COMMENTED** [review]
also might wanna consider renaming it to `Projucer`, because it is always installed as uppercase

**Comment by ecrax** [comment]
thanks for the suggestions :)

**Review by kbdharun — APPROVED** [review]
LGTM

File: pages/common/projucer.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# projucer
```
Comment (pixelcmtd) [inline pages/common/projucer.md:1]:
```suggestion
# Projucer
```

Comment (kbdharun) [reply pages/common/projucer.md:1]: Just curious, Will there be linter errors for having Uppercase in the heading?

Comment (pixelcmtd) [reply pages/common/projucer.md:1]: There won't.

a screenshot showing that it works fine

Comment (kbdharun) [reply pages/common/projucer.md:1]: > There won't.
> 
> a screenshot showing that it works fine
> 

Thanks 👍🏻

**Review by pixelcmtd — APPROVED** [review]

File: pages/common/projucer.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Generate a JUCE project from a PIP file:
+
+`Projucer --create-project-from-pip {{path/to/PIP}} {{path/to/output}}`
+
+- Remove all JUCE-style comments (`//=====`, `//-----` or `///////`):
+
+`Projucer --tidy-divider-comments {{path/to/target_folder}}`
+
+- Show help:
```
Comment (CleanMachine1) [inline pages/common/projucer.md:26]:
```suggestion
- Display help:
```
Just sticking with the standard.

lgtm after this.

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**